presto-server-rpm/pom.xml

```diff
@@ -169,12 +205,96 @@
                                 <rule>
                                     <destination>/usr/lib/presto/lib</destination>
                                 </rule>
-
                             </rules>
                         </package>
                     </packages>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
```

Comment:
That's cool, but does it handle docker-compose -style setups?
eg if we wanted to smoke test RPM with a connector?

Reply:
Looks like it: https://dmp.fabric8.io/#docker-compose

I was thinking (after I got this all working) that doing this with `DockerContainer` might be better, but this works for now.